src/System.Data.SqlClient/tests/ManualTests/SQL/MirroringTest/ConnectionOnMirroringTest.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public static class ConnectionOnMirroringTest
+    {
+        [CheckConnStrSetupFact]
+        public static void TestMultipleConnectionToMirroredServer()
+        {
+            string connectionString = DataTestUtility.TcpConnStr;
+            int mirroringState;
+            string failoverPartnerName;
+            bool isMirroring = GetMirroringInfo(connectionString, out mirroringState, out failoverPartnerName);
+            List<SqlConnection> list = new List<SqlConnection>();
+            if (isMirroring && mirroringState == 4 && !string.IsNullOrEmpty(failoverPartnerName))
```

Comment:
Is there any enum in SqlClient which better describes that mirroring state number?

Reply:
If not, can you create a named variable which describes what the state number means?